aws-lc-sys/symbols/x86_64-unknown-linux-gnu.txt

```diff
@@ -3717,6 +3731,151 @@ poly_Rq_mul
 pq_custom_randombytes
 pq_custom_randombytes_init_for_testing
 pq_custom_randombytes_use_deterministic_for_testing
+pqcrystals_kyber1024_ref_barrett_reduce
```

Comment:
It's surprising all of these symbols are public. Can any of them be marked as `static` upstream?

Reply:
I believe we are using the reference Kyber implementation, which we've not diverged from.